doc/Language/variables.pod6

```diff
@@ -1532,6 +1532,9 @@ It stringifies to C<Raku>:
 
     $*RAKU.put; # OUTPUT: «Raku␤»
 
+B<Note:> Before Rakudo version 2020.1, this information was available
+through the C<$*PERL> variable.
+
 X<|$*PERL>
 =head4 C<$*PERL>
```

Comment:
It still is. We cam maybe discourage its use, but it's still there.

Reply:
I mentioned both are now available, for the time being.